material/utils.py

```diff
@@ -0,0 +1,30 @@
+import os
+import numpy as np
+from PIL import Image
+from fnmatch import fnmatch
+
+
+def clear_folder(path, unwanted_resolution=(1, 1, 3)):
```

Comment:
"clear_folder" sounds to me like your removing everything that's in the folder. Can we make the function name a bit more descriptive? Like "remove_low_resolution_images" or at least "clean" instead of "clear".

Also, are we sure there aren't any images with, for example, 2x2 size or something? I guess it would be a bit more general if we removed everything that is smaller than a certain number of pixels. 

Example: change change the unwanted_resolution parameter to min_resolution=100 (or any other value) and then check for temp_shape[0] > min_resolution and temp_shape[1] > min_resolution. This way, we are sure there are no other small pictures anymore and it is more general for use in other competitions.